### PR TITLE
Fix part of #103: Fix lint issue in spec package

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix part of #103 
Fixed `.js files in spec` package

## Reviewer 
@jameesjohn 

## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: spec/utilsSpec.js [hunk 1]
```diff
@@ -171,9 +171,10 @@ describe('Utility module tests', () => {
     firstItemLink = 'OppiabotTestActivitiesModel'.link(
       firstModelFileObj.blob_url
     );
-    let secondItemLink = 'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'.link(
-      secondModelFileObj.blob_url
-    );
+    let secondItemLink =
+      'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'.link(
+        secondModelFileObj.blob_url
+      );
```
Comment (vojtechjelinek) [inline spec/utilsSpec.js:177]:
```suggestion
    let secondItemLink = (
      'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'.link(
        secondModelFileObj.blob_url
      )
    );
```

File: spec/setup.js [hunk 2]
```diff
@@ -165,8 +165,8 @@ runTest().then(
...
-  }).catch(() => {
-    revertEnv();
-    console.log('Tests failed.')
-    process.exit(1);
-  });
+  })['catch'](() => {
+  revertEnv();
+  console.log('Tests failed.');
+  process.exit(1);
+});
```
Comment (vojtechjelinek) [inline spec/setup.js:172]: Why this change?

File: spec/periodicChecksSpec.js [hunk 3]
```diff
@@ -426,70 +426,70 @@ describe('Periodic Checks Module', () => {
...
+      it('should ping pr author', () => {
+        expect(github.issues.createComment).toHaveBeenCalled();
+        expect(github.issues.createComment).toHaveBeenCalledWith({
+          issue_number: 4,
+          owner: 'oppia',
+          repo: 'oppia',
+          body:
+            'Hi @author4, this PR is ready to be merged. Please address any ' +
+            'remaining comments prior to merging, and feel free to merge ' +
+            "this PR once the CI checks pass and you're happy with it. Thanks!",
```
Comment (vojtechjelinek) [inline spec/periodicChecksSpec.js:467]:
```suggestion
            'this PR once the CI checks pass and you\'re happy with it. Thanks!',
```

File: spec/periodicChecksSpec.js [hunk 4]
```diff
@@ -129,15 +129,15 @@ describe('Periodic Checks Module', () => {
   };
 
   beforeEach(async () => {
-    spyOn(scheduler, 'createScheduler').and.callFake(() => {});
+    spyOn(scheduler, 'createScheduler').and.callFake(() => { });
 
     github = {
       issues: {
         createComment: jasmine
```
Comment (vojtechjelinek) [inline spec/periodicChecksSpec.js:136]:
```suggestion
        createComment: jasmine.createSpy('createComment').and.callFake(() => {})
```

File: spec/checkWipDraftPRSpec.js [hunk 5]
```diff
@@ -43,17 +45,19 @@ describe('Oppiabot\'s', () => {
...
-    spyOn(checkCriticalPullRequestModule, 'checkIfPRAffectsDatastoreLayer').and.callFake(() => {});
-    spyOn(checkPullRequestTemplateModule,'checkTemplate').and.callFake(() => {});
-    spyOn(newCodeOwnerModule, 'checkForNewCodeowner').and.callFake(() => {});
+      .and.callFake(() => { });
+    spyOn(checkPullRequestJobModule, 'checkForNewJob').and.callFake(() => { });
+    spyOn(checkCriticalPullRequestModule, 'checkIfPRAffectsDatastoreLayer')
+      .and.callFake(() => { });
+    spyOn(checkPullRequestTemplateModule, 'checkTemplate')
+      .and.callFake(() => { });
+    spyOn(newCodeOwnerModule, 'checkForNewCodeowner').and.callFake(() => { });
```
Comment (vojtechjelinek) [inline spec/checkWipDraftPRSpec.js:60]:
```suggestion
    ).and.callFake(() => { });
    spyOn(checkPullRequestJobModule, 'checkForNewJob').and.callFake(() => { });
    spyOn(
      checkCriticalPullRequestModule, 'checkIfPRAffectsDatastoreLayer'
    ).and.callFake(() => { });
    spyOn(
      checkPullRequestTemplateModule, 'checkTemplate'
    ).and.callFake(() => { });
    spyOn(newCodeOwnerModule, 'checkForNewCodeowner').and.callFake(() => { });
```

File: spec/checkPullRequestTemplateSpec.js [hunk 6]
```diff
@@ -377,79 +379,82 @@ describe('Pull Request Template', () => {
...
-          'Thanks!',
+      it('should ping PR author', () => {
+        expect(github.issues.createComment).toHaveBeenCalled();
+        expect(github.issues.createComment).toHaveBeenCalledWith({
+          issue_number: payloadData.payload.pull_request.number,
+          repo: payloadData.payload.repository.name,
+          owner: payloadData.payload.repository.owner.login,
+          body:
+            'Hi @' +
+            payloadData.payload.pull_request.user.login +
```
Comment (vojtechjelinek) [inline spec/checkPullRequestTemplateSpec.js:441]:
```suggestion
            'Hi @' + payloadData.payload.pull_request.user.login +
```

File: spec/checkPullRequestJobSpec.js [hunk 7]
```diff
@@ -21,8 +21,11 @@ const apiForSheetsModule = require('../lib/apiForSheets');
 const checkPullRequestLabelsModule = require('../lib/checkPullRequestLabels');
 const checkPullRequestBranchModule = require('../lib/checkPullRequestBranch');
 const checkWIPModule = require('../lib/checkWipDraftPR');
-const checkCriticalPullRequestModule = require('../lib/checkCriticalPullRequest');
-const checkPullRequestTemplateModule = require('../lib/checkPullRequestTemplate');
+const checkCriticalPullRequestModule =
+  require('../lib/checkCriticalPullRequest');
+const checkPullRequestTemplateModule =
+
```
Comment (vojtechjelinek) [inline spec/checkPullRequestJobSpec.js:27]: Remove this empty line.

File: spec/checkPullRequestJobSpec.js [hunk 8]
```diff
@@ -227,16 +453,18 @@ describe('Pull Request Job Spec', () => {
...
-    spyOn(checkPullRequestTemplateModule,'checkTemplate').and.callFake(() => {});
-    spyOn(newCodeOwnerModule, 'checkForNewCodeowner').and.callFake(() => {});
+    ).and.callFake(() => { });
+    spyOn(checkCriticalPullRequestModule, 'checkIfPRAffectsDatastoreLayer')
+      .and.callFake(() => { });
+    spyOn(checkPullRequestBranchModule, 'checkBranch').and.callFake(() => { });
+    spyOn(checkWIPModule, 'checkWIP').and.callFake(() => { });
+    spyOn(checkPullRequestTemplateModule, 'checkTemplate')
+      .and.callFake(() => { });
+    spyOn(newCodeOwnerModule, 'checkForNewCodeowner').and.callFake(() => { });
```
Comment (vojtechjelinek) [inline spec/checkPullRequestJobSpec.js:467]:
```suggestion
    spyOn(
      checkCriticalPullRequestModule, 'checkIfPRAffectsDatastoreLayer'
    ).and.callFake(() => { });
    spyOn(checkPullRequestBranchModule, 'checkBranch').and.callFake(() => { });
    spyOn(checkWIPModule, 'checkWIP').and.callFake(() => { });
    spyOn(
      checkPullRequestTemplateModule, 'checkTemplate'
    ).and.callFake(() => { });
    spyOn(newCodeOwnerModule, 'checkForNewCodeowner').and.callFake(() => { });
```

File: spec/checkPullRequestJobSpec.js [hunk 9]
```diff
@@ -271,18 +499,23 @@ describe('Pull Request Job Spec', () => {
           'https://github.com/oppia/oppia/wiki/Running-jobs-in-production' +
           '#submitting-a-pr-with-a-new-job'));
       const jobRegistryLink = 'job registry'.link(
-        'https://github.com/oppia/oppia/blob/develop/core/jobs_registry.py');
-      const jobNameLink = 'FirstTestOneOffJob'.link(firstNewJobFileObj.blob_url);
+        'https://github.com/oppia/oppia/blob/develop/core/jobs_registry.py'
+      );
+      const jobNameLink = 'FirstTestOneOffJob'
+        .link(firstNewJobFileObj.blob_url);
```
Comment (vojtechjelinek) [inline spec/checkPullRequestJobSpec.js:505]:
```suggestion
      const jobNameLink = (
        'FirstTestOneOffJob'.link(firstNewJobFileObj.blob_url));
```

Comment (vojtechjelinek) [reply spec/checkPullRequestJobSpec.js:505]: Similar changes below

File: spec/checkPullRequestJobSpec.js [hunk 10]
```diff
@@ -644,7 +889,8 @@ describe('Pull Request Job Spec', () => {
 
   describe('When pull request has datastore label', () => {
     beforeEach(async () => {
-      payloadData.payload.pull_request.labels = [{ name: 'PR: Affects datastore layer' }];
+      payloadData.payload.pull_request.labels =
+        [{ name: 'PR: Affects datastore layer' }];
```
Comment (vojtechjelinek) [inline spec/checkPullRequestJobSpec.js:893]:
```suggestion
      payloadData.payload.pull_request.labels = [{ 
        name: 'PR: Affects datastore layer' 
      }];
```

File: spec/checkPullRequestLabelsSpec.js [hunk 11]
```diff
@@ -383,11 +385,12 @@ describe('Pull Request Label Check', () => {
     beforeEach(async () => {
       payloadData.payload.action = 'labeled';
       payloadData.payload.label = label;
-      spyOn(checkPullRequestLabelModule, 'checkForIssueLabel').and.callThrough();
+      spyOn(checkPullRequestLabelModule, 'checkForIssueLabel')
+        .and.callThrough();
```
Comment (vojtechjelinek) [inline spec/checkPullRequestLabelsSpec.js:389]:
```suggestion
      spyOn(
        checkPullRequestLabelModule, 'checkForIssueLabel'
      ).and.callThrough();
```

File: spec/checkPullRequestLabelsSpec.js [hunk 12]
```diff
@@ -433,11 +435,12 @@ describe('Pull Request Label Check', () => {
...
```
Comment (vojtechjelinek) [inline spec/checkPullRequestLabelsSpec.js:439]:
```suggestion
      spyOn(
        checkPullRequestLabelModule, 'checkForIssueLabel'
      ).and.callThrough();
```

File: spec/checkPullRequestLabelsSpec.js [hunk 13]
```diff
@@ -462,7 +465,8 @@ describe('Pull Request Label Check', () => {
     beforeEach(async () => {
       payloadData.payload.action = 'unlabeled';
       payloadData.payload.label = label;
-      spyOn(checkPullRequestLabelModule, 'checkCriticalLabel').and.callThrough();
+      spyOn(checkPullRequestLabelModule, 'checkCriticalLabel')
+        .and.callThrough();
```
Comment (vojtechjelinek) [inline spec/checkPullRequestLabelsSpec.js:469]:
```suggestion
      spyOn(
        checkPullRequestLabelModule, 'checkCriticalLabel'
      ).and.callThrough();
```

File: spec/checkPullRequestLabelsSpec.js [hunk 14]
```diff
@@ -508,11 +513,12 @@ describe('Pull Request Label Check', () => {
       payloadData.payload.action = 'unlabeled';
       payloadData.payload.label = label;
       payloadData.payload.sender.login = 'seanlip';
-      spyOn(checkPullRequestLabelModule, 'checkCriticalLabel').and.callThrough();
+      spyOn(checkPullRequestLabelModule, 'checkCriticalLabel')
+        .and.callThrough();
```
Comment (vojtechjelinek) [inline spec/checkPullRequestLabelsSpec.js:517]:
```suggestion
      spyOn(
        checkPullRequestLabelModule, 'checkCriticalLabel'
      ).and.callThrough();
```

File: spec/checkPullRequestLabelsSpec.js [hunk 15]
```diff
@@ -538,11 +543,12 @@ describe('Pull Request Label Check', () => {
     beforeEach(async () => {
       payloadData.payload.action = 'unlabeled';
       payloadData.payload.label = label;
-      spyOn(checkPullRequestLabelModule, 'checkCriticalLabel').and.callThrough();
+      spyOn(checkPullRequestLabelModule, 'checkCriticalLabel')
+        .and.callThrough();
```
Comment (vojtechjelinek) [inline spec/checkPullRequestLabelsSpec.js:547]:
```suggestion
      spyOn(
        checkPullRequestLabelModule, 'checkCriticalLabel'
      ).and.callThrough();
```

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Left a few comments.

File: spec/checkMergeConflictSpec.js [hunk 16]
```diff
@@ -51,7 +51,7 @@ describe('Merge Conflict Check', () => {
       "PR:%20don't%20merge%20-%20HAS%20MERGE%20CONFLICTS",
     name: "PR: don't merge - HAS MERGE CONFLICTS",
     color: 'd93f0b',
-    default: false,
+    'default': false,
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply spec/checkMergeConflictSpec.js:54]: error  Unquoted reserved word 'default' used as key  quote-props

File: spec/utilsSpec.js (hunk 1 above)
Comment (anandwana001) [reply spec/utilsSpec.js:177]: Why this change?
It could be:
```
let secondItemLink =
      'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'
        .link(secondModelFileObj.blob_url);
```

File: spec/setup.js (hunk 2 above)
Comment (anandwana001) [reply spec/setup.js:172]: error  .catch is a syntax error  dot-notation

File: spec/periodicChecksSpec.js (hunk 4 above)
Comment (anandwana001) [reply spec/periodicChecksSpec.js:136]: this will give max-len issue

**Comment by anandwana001** [comment]
@vojtechjelinek @jameesjohn mentioning for further review.

File: spec/checkPullRequestJobSpec.js [hunk 17]
```diff
@@ -478,25 +721,29 @@ describe('Pull Request Job Spec', () => {
     it('should ping server jobs admin', () => {
       expect(github.issues.createComment).toHaveBeenCalled();
       const author = payloadData.payload.pull_request.user.login;
-      const formText = 'server jobs form'.link(
-        'https://goo.gl/forms/XIj00RJ2h5L55XzU2');
+      const formText = (
+        'server jobs form'.link('https://goo.gl/forms/XIj00RJ2h5L55XzU2'));
```
Comment (jameesjohn) [inline spec/checkPullRequestJobSpec.js:725]: Just curious, was this a recommended change from eslint?

File: spec/checkPullRequestJobSpec.js [hunk 18]
```diff
@@ -478,25 +721,29 @@ describe('Pull Request Job Spec', () => {
...
           'https://github.com/oppia/oppia/wiki/Running-jobs-in-production' +
           '#submitting-a-pr-with-a-new-job'));
-      const jobRegistryLink = 'job registry'.link(
-        'https://github.com/oppia/oppia/blob/develop/core/jobs_registry.py');
-      const jobNameLink = 'OppiabotContributionsOneOffJob'.link(modifiedExistingJobFileObj.blob_url);
+      const jobRegistryLink = (
+        'job registry'.link(
+          'https://github.com/oppia/oppia/blob/develop/core/jobs_registry.py'));
+      const jobNameLink = 'OppiabotContributionsOneOffJob'.
+        link(modifiedExistingJobFileObj.blob_url);
```
Comment (jameesjohn) [inline spec/checkPullRequestJobSpec.js:735]: We should either do what was done on line 731 or change this to 
```
const jobNameLink = 'OppiabotContributionsOneOffJob'.link(
  modifiedExistingJobFileObj.blob_url
);
```

File: spec/checkPullRequestReviewSpec.js [hunk 19]
```diff
@@ -649,347 +651,351 @@ describe('Pull Request Review Module', () => {
...
-            "once the CI checks pass and you're happy with it. Thanks!",
+        it('should ping pr author', () => {
+          expect(github.issues.createComment).toHaveBeenCalled();
+          expect(github.issues.createComment).toHaveBeenCalledWith({
+            owner: reviewPayloadData.payload.repository.owner.login,
+            repo: reviewPayloadData.payload.repository.name,
+            issue_number: reviewPayloadData.payload.pull_request.number,
+            body:
+              'Hi @' +
+              reviewPayloadData.payload.pull_request.user.login +
```
Comment (jameesjohn) [inline spec/checkPullRequestReviewSpec.js:879]: These two lines can go on one line, right?

File: spec/checkPullRequestTemplateSpec.js [hunk 20]
```diff
@@ -377,79 +379,81 @@ describe('Pull Request Template', () => {
...
-          'to add this by ticking on the check box. Thanks!',
+      it('should ping PR author', () => {
+        expect(github.issues.createComment).toHaveBeenCalled();
+        expect(github.issues.createComment).toHaveBeenCalledWith({
+          issue_number: payloadData.payload.pull_request.number,
+          repo: payloadData.payload.repository.name,
+          owner: payloadData.payload.repository.owner.login,
+          body:
+            'Hi @' +
+            payloadData.payload.pull_request.user.login +
```
Comment (jameesjohn) [inline spec/checkPullRequestTemplateSpec.js:402]: Ditto

**Comment by vojtechjelinek** [comment]
@anandwana001 Please respond with 'Done' to all the comments that I have left and you have fixed. I will do a second pass later today.

**Review by jameesjohn — CHANGES_REQUESTED** [review]
Thanks, @anandwana001.
I left a few comments PTAL, thanks!

File: spec/apiForSheetsSpec.js [hunk 21]
```diff
@@ -110,7 +114,7 @@ describe('Api For Sheets Module', () => {
           values: {
             get: jasmine.createSpy('get').and.callFake(async (obj, cb) => {
               await cb(null, {
-                data : {values: [['test']]},
+                data: { values: [['test']] },
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply spec/apiForSheetsSpec.js]: Done

File: spec/apiForSheetsSpec.js [hunk 22]
```diff
@@ -243,40 +248,42 @@ describe('Api For Sheets Module', () => {
...
+        expect(github.issues.createComment).toHaveBeenCalledWith({
+          owner: pullRequestPayload.payload.repository.owner.login,
+          repo: pullRequestPayload.payload.repository.name,
+          number: pullRequestPayload.payload.pull_request.number,
+          body:
+            'Hi! @testuser7778,' +
+            ' Welcome to Oppia! Please could you ' +
+            'follow the instructions ' +
+            linkResult +
+            " to get started? You'll need to do " +
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply spec/apiForSheetsSpec.js]: Done

File: spec/apiForSheetsSpec.js [hunk 23]
```diff
@@ -300,15 +307,15 @@ describe('Api For Sheets Module', () => {
           values: {
             get: jasmine.createSpy('get').and.callFake(async (obj, cb) => {
               await cb(null, {
-                data : {values: [['test']]},
+                data: { values: [['test']] },
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply spec/apiForSheetsSpec.js]: Done

File: spec/apiForSheetsSpec.js [hunk 24]
```diff
@@ -336,10 +344,12 @@ describe('Api For Sheets Module', () => {
...
 
-    it('should be called with the correct username for the given payload', () => {
-      const context = apiForSheetsModule.checkClaStatus.calls.argsFor(0)[0];
-      expect(context.payload.pull_request.user.login).toEqual('testuser7777');
-    });
+    it('should be called with the correct username for the given payload',
+      () => {
+        const context = apiForSheetsModule.checkClaStatus.calls.argsFor(0)[0];
+        expect(context.payload.pull_request.user.login)
+          .toEqual('testuser7777');
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply spec/apiForSheetsSpec.js]: Done

File: spec/checkCriticalPullRequestSpec.js [hunk 25]
```diff
@@ -192,18 +195,20 @@ describe('Critical Pull Request Spec', () => {
...
-    spyOn(checkCriticalPullRequestModule, 'checkIfPRAffectsDatastoreLayer').and.callThrough();
+    ).and.callFake(() => { });
+    spyOn(checkPullRequestBranchModule, 'checkBranch').and.callFake(() => { });
+    spyOn(checkWIPModule, 'checkWIP').and.callFake(() => { });
+    spyOn(checkPullRequestTemplateModule, 'checkTemplate')
+      .and.callFake(() => { });
+    spyOn(newCodeOwnerModule, 'checkForNewCodeowner').and.callFake(() => { });
+
+    spyOn(checkCriticalPullRequestModule, 'checkIfPRAffectsDatastoreLayer')
+      .and.callThrough();
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply spec/checkCriticalPullRequestSpec.js]: Done

File: spec/checkCriticalPullRequestSpec.js [hunk 26]
```diff
@@ -218,7 +223,8 @@ describe('Critical Pull Request Spec', () => {
     });
 
     it('should check for models', () => {
-      expect(checkCriticalPullRequestModule.checkIfPRAffectsDatastoreLayer).toHaveBeenCalled();
+      expect(checkCriticalPullRequestModule.checkIfPRAffectsDatastoreLayer)
+        .toHaveBeenCalled();
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply spec/checkCriticalPullRequestSpec.js]: Done

File: spec/checkCriticalPullRequestSpec.js [hunk 27]
```diff
@@ -274,7 +280,8 @@ describe('Critical Pull Request Spec', () => {
...
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply spec/checkCriticalPullRequestSpec.js]: Done

File: spec/checkCriticalPullRequestSpec.js [hunk 28]
```diff
@@ -286,9 +293,11 @@ describe('Critical Pull Request Spec', () => {
...
         newModelFileObj.blob_url
       );
-      const secondModels = 'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'.link(
-        modifiedModelFileObj.blob_url
-      );
+      const secondModels =
+        'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'
+          .link(
+            modifiedModelFileObj.blob_url
+          );
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply spec/checkCriticalPullRequestSpec.js]: Done

File: spec/checkPullRequestJobSpec.js (hunk 17 above)
Comment (anandwana001) [reply spec/checkPullRequestJobSpec.js:725]: https://github.com/oppia/oppiabot/pull/148#discussion_r511668220

File: spec/periodicChecksSpec.js (hunk 3 above)
Comment (anandwana001) [reply spec/periodicChecksSpec.js]: Done

File: spec/checkWipDraftPRSpec.js (hunk 5 above)
Comment (anandwana001) [reply spec/checkWipDraftPRSpec.js]: Done

File: spec/checkPullRequestTemplateSpec.js (hunk 6 above)
Comment (anandwana001) [reply spec/checkPullRequestTemplateSpec.js]: Done

File: spec/checkPullRequestJobSpec.js (hunk 7 above)
Comment (anandwana001) [reply spec/checkPullRequestJobSpec.js]: Done

File: spec/checkPullRequestJobSpec.js (hunk 8 above)
Comment (anandwana001) [reply spec/checkPullRequestJobSpec.js]: Done

File: spec/checkPullRequestJobSpec.js (hunk 9 above)
Comment (anandwana001) [reply spec/checkPullRequestJobSpec.js]: Done

File: spec/checkPullRequestJobSpec.js (hunk 10 above)
Comment (anandwana001) [reply spec/checkPullRequestJobSpec.js]: Done

File: spec/checkPullRequestLabelsSpec.js (hunk 11 above)
Comment (anandwana001) [reply spec/checkPullRequestLabelsSpec.js]: Done

File: spec/checkPullRequestLabelsSpec.js (hunk 12 above)
Comment (anandwana001) [reply spec/checkPullRequestLabelsSpec.js]: Done

File: spec/checkPullRequestLabelsSpec.js (hunk 13 above)
Comment (anandwana001) [reply spec/checkPullRequestLabelsSpec.js]: Done

File: spec/checkPullRequestLabelsSpec.js (hunk 14 above)
Comment (anandwana001) [reply spec/checkPullRequestLabelsSpec.js]: Done

File: spec/checkPullRequestLabelsSpec.js (hunk 15 above)
Comment (anandwana001) [reply spec/checkPullRequestLabelsSpec.js]: Done

File: spec/checkPullRequestJobSpec.js (hunk 18 above)
Comment (anandwana001) [reply spec/checkPullRequestJobSpec.js:735]: @vojtechjelinek

File: spec/periodicChecksSpec.js (hunk 4 above)
Comment (vojtechjelinek) [reply spec/periodicChecksSpec.js:136]: What is the length limit, since this line is 80 chars and I though we allow max 80 chars (<=80)

**Review by vojtechjelinek — APPROVED** [review]
LGTM! Just one nit.

File: spec/checkMergeConflictSpec.js (hunk 16 above)
[the remark this answers is not in the document]
Comment (vojtechjelinek) [reply spec/checkMergeConflictSpec.js:54]: Hmm, okay, then maybe quote all the keys in this dictionary.

File: spec/utilsSpec.js (hunk 1 above)
Comment (vojtechjelinek) [reply spec/utilsSpec.js]: Yeah, I guess this is also fine, I prefer to use (even when the second `()` are not needed). I think the final decision is up to you.

```typescript
let secondItemLink = (
  'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'
    .link(secondModelFileObj.blob_url);
);
```

File: spec/periodicChecksSpec.js (hunk 4 above)
Comment (anandwana001) [reply spec/periodicChecksSpec.js:136]: we allow max 80, and this line is going over.

File: spec/checkMergeConflictSpec.js (hunk 16 above)
[the remark this answers is not in the document]
Comment (anandwana001) [reply spec/checkMergeConflictSpec.js:54]: we can't do that too because of below error:
```bash
error  Unnecessarily quoted property 'id' found       quote-props
```

File: spec/checkPullRequestTemplateSpec.js (hunk 20 above)
Comment (anandwana001) [reply spec/checkPullRequestTemplateSpec.js:402]: Done

File: spec/checkPullRequestReviewSpec.js (hunk 19 above)
Comment (anandwana001) [reply spec/checkPullRequestReviewSpec.js:879]: Done

File: spec/checkPullRequestJobSpec.js (hunk 17 above)
Comment (jameesjohn) [reply spec/checkPullRequestJobSpec.js:725]: Okay.

File: spec/checkPullRequestJobSpec.js [hunk 29]
```diff
@@ -478,25 +721,31 @@ describe('Pull Request Job Spec', () => {
...
-      const jobRegistryLink = 'job registry'.link(
-        'https://github.com/oppia/oppia/blob/develop/core/jobs_registry.py');
-      const jobNameLink = 'OppiabotContributionsOneOffJob'.link(modifiedExistingJobFileObj.blob_url);
+      const jobRegistryLink = (
+        'job registry'.link(
+          'https://github.com/oppia/oppia/blob/develop/core/jobs_registry.py'));
+      const jobNameLink =
+        'OppiabotContributionsOneOffJob'.link(
+          modifiedExistingJobFileObj.blob_url
+        );
```
Comment (jameesjohn) [inline spec/checkPullRequestJobSpec.js:737]: Modify this to include the parens as was suggested by @vojtechjelinek https://github.com/oppia/oppiabot/pull/148#discussion_r511668220

File: spec/checkPullRequestJobSpec.js (hunk 18 above)
Comment (jameesjohn) [reply spec/checkPullRequestJobSpec.js]: I don't think so. I'm saying we should choose one style, and use that in all places.

File: spec/checkPullRequestJobSpec.js (hunk 29 above)
Comment (anandwana001) [reply spec/checkPullRequestJobSpec.js]: Done

File: spec/checkPullRequestJobSpec.js (hunk 18 above)
Comment (anandwana001) [reply spec/checkPullRequestJobSpec.js]: Done

**Review by jameesjohn — APPROVED** [review]
LGTM

**Comment by jameesjohn** [comment]
Thanks, @anandwana001 for this PR!
